### ISSUE TITLE
[SIN ]Pre-process the titanic dataset and implement different Machine Learning algorithms

### ISSUE THREAD

**Issue by AjayKhalsa** [CMPN-CODECELL/Hacktoberfest2020#7]
Tasks
- Clean and filter the titanic dataset.
- Implement ML algorithms like
   A few example solutions:
    - Logistic Regression

    - Random Forest Classifier

    - SVM Classifier

    - Decision Tree

    - K-Means

    - Gradient Boosting (XGBoost, CatBoost)

    - Neural networks

    - Other
